pkg/pods/pod.go

```diff
@@ -651,3 +651,20 @@ func (p *Pod) logExec() runit.LogExec {
 	}
 	return p.LogExec
 }
+
+// LogBridgeExec returns the full path to successfully execute LogBridge
+// This slice is eventually serialized into YAML, so we're careful about escaping
+func (p *Pod) LogBridgeExec() []string {
+	return []string{
+		"/usr/bin/chpst",
+		"-u",
+		"nobody",
+		"-e",
+		p.EnvDir(),
+		"--",
+		filepath.Join(DEFAULT_PATH, "p2-preparer", "p2-log-bridge", "current", "bin", "p2-log-bridge"),
```

Comment:
The pods package has generally been decoupled from the preparer (a specific implementation of a thing that launches pods). You can imagine that this would break clients who don't use a preparer, ie folks that just rely on `p2-launch` or some alternative.

Instead of hardcoding preparer values here, I'd suggest putting them in `pkg/preparer` and injecting the logexec command into a field in `pods.Pod`. Additionally, it'd be cleaner to read the pod home from the environment instead of assuming the default path - that makes it a lot easier to do per-pod testing of regular `svlogd` vs `p2-log-bridge` as well.